database/factories/ModelFactory.php

```diff
@@ -52,7 +51,7 @@
         'user_id'       => $faker->randomElement($userIds ?: $userIds = App\Models\User::pluck('id')->toArray()),
         'title'         => $faker->name,
         'description'   => $faker->text($maxNbChars = 500),
-        'type_id'       => $faker->randomElement($typeId ?: $typeId = App\Models\Type::pluck('id')->toArray()),
+        'category_id'   => $faker->randomElement($typeId ?: $typeId = App\Models\Type::pluck('id')->toArray()),
```

Comment:
change to `$faker->randomElement($typeId ?: $typeId = App\Models\Category::pluck('id')->toArray()),`